frontend/src/pages/offers/PalletCard.tsx

```diff
@@ -0,0 +1,57 @@
+import cx from 'classnames'
+import { FunctionComponent } from 'react'
+import { PalletConfig } from '../../data/constants'
+import { PalletType } from '../../types/api-types'
+
+interface Props extends PalletConfig {
+  isSelected?: boolean
+  onClick: (type: PalletType) => void
+}
+
+/**
+ * A fully-controlled wrapper around a <radio> input that allows users to select
+ * a pallet card within the CreatePalletModal.
+ */
+const PalletCard: FunctionComponent<Props> = (props) => {
```

Comment:
This is an individual row in the `CreatePalletModal`. It's not the easiest thing to read BUT it's fully accessible for keyboard users SO.